vllm/v1/attention/backends/flash_attn.py

```diff
@@ -205,10 +205,11 @@ def unified_v1_flash_attention(
         v_scale,
     )
 
-    attn_output = flash_attn_varlen_func(
+    flash_attn_varlen_func(
```

Comment:
Just a nit for readability
```suggestion
    # Compute attention and update output up to `num_actual_tokens`
    flash_attn_varlen_func(
```

Reply:
Sounds good. Updated.